token_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestToken(t *testing.T) {
+func TestTokenAuthPassword(t *testing.T) {
```

Comment:
Add test for other methods?

Reply:
Yes. For Password and Application Credential.